video-app/src/app/shared/control-panel/control-panel.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, EventEmitter, Output, Input } from '@angular/core';
+import { SortEnum } from '@core/enums/sort.enum';
+
+@Component({
+  selector: 'app-control-panel',
+  templateUrl: './control-panel.component.html',
+  styleUrls: ['./control-panel.component.less'],
+})
+export class ControlPanelComponent {
+  @Input() public sort: SortEnum;
+  @Input() public fav: boolean;
+  @Input() public view: string;
+
+  @Output() public sortChange = new EventEmitter();
+  @Output() public viewChange = new EventEmitter();
+  @Output() public favChange = new EventEmitter();
+  @Output() public loadVideoEvent = new EventEmitter();
```

Comment:
w jakim celu to jest emitowane?

Reply:
two-way binding do propertisów sort, fav, view